docs/guides/volumesnapshot/pvc/index.md

```diff
@@ -135,263 +140,406 @@ spec:
-pvc-volume-snapshot-fnbwz        BackupConfiguration   pvc-volume-snapshot   Succeeded   1m32s
+Every 2.0s: kubectl get snapshots.storage.kubestash.com -n demo -l=kubestash.com/repo-name=gcs-repository                                   workstation: Tue Jan 16 17:03:06 2024
+
+NAME                                                            REPOSITORY       SESSION           SNAPSHOT-TIME          DELETION-POLICY   PHASE       VERIFICATION-STATUS   AGE
+gcs-repository-pvc-volume-snapshot-frequent-backup-1705402801   gcs-repository   frequent-backup   2024-01-16T11:00:08Z   Delete            Succeeded                         3m6s
 ```
 
-We can see above that the backup session has succeeded. Now, we are going to verify that the `VolumeSnapshot` has been created and the snapshots have been stored in the respective backend.
+> When a backup is triggered according to schedule, KubeStash will create a `Snapshot` with the following labels  `kubestash.com/app-ref-kind: PersistentVolumeClaim`, `kubestash.com/app-ref-name: <volume-name>`, `kubestash.com/app-ref-namespace: <volume-namespace>` and `kubestash.com/repo-name: <repository-name>`. We can use these labels to watch only the `Snapshot` of our desired Workload or `Repository`.
+
+Now, retrieve the YAML representation of the above created `Snapshot`, and inspect the `spec.status` section to see the backup information of the targeted backup components.
+
+Now, retrieve the YAML representation of the above created `Snapshot`, and inspect the `spec.status` section to see the backup information of the targeted backup components.
+
+```bash
+$ kubectl get snapshots  -n demo gcs-repository-pvc-volume-snapshot-frequent-backup-1705402801 -o yaml
+```
+
+```yaml
+apiVersion: storage.kubestash.com/v1alpha1
+kind: Snapshot
+metadata:
+  name: gcs-repository-pvc-volume-snapshot-frequent-backup-1705402801
+  namespace: demo
+spec:
+  ---
+status:
+  components:
+    volumesnapshot:
+      driver: VolumeSnapshotter
+      phase: Succeeded
+      volumeSnapshotterStats:
+        - pvcName: source-pvc
+          volumeSnapshotName: source-pvc-1705402801
+  conditions:
+    ---
+  phase: Succeeded
+  snapshotTime: "2024-01-16T11:00:08Z"
+  totalComponents: 1
+```
+
+Here,
+- `spec.status.components` specifies the backup information of the backup target components.
+- `spec.status.components.[<component-name>].volumeSnapshotterStats` specifies the information about the `VolumeSnapshotter` driver. In the case of a standalone `PVC`, the component name is designated as `volumesnapshot`.
+  - `volumeSnapshotterStats.hostPath` specifies the corresponding path of `PVC` volume for which `volumeSnapshot` has created.
+  - `volumeSnapshotterStats.pvcName` specifies the name of the backed-up target `PVC` volume.
+  - `volumeSnapshotterStats.volumeSnapshotName` specifies the created `VolumeSnapshot` object name corresponding to the target `PVC` volume.
 
```

Comment:
KubeStash keeps the backup for Snapshot YAMLs. If we navigate to the GCS bucket, we will see the Snapshot YAML stored in the <> directory.

Reply:
> applies to all the relevant places